samples/graphene-nginx/README.md

```diff
@@ -1,9 +1,13 @@
 # Graphene nginx sample
-This sample shows how to modify the [Graphene nginx sample](https://github.com/oscarlab/graphene/tree/master/Examples/nginx) to run in Marblerun.
+This sample is a slightly modified [Graphene nginx sample](https://github.com/oscarlab/graphene/tree/master/Examples/nginx) to run in Marblerun.
 
-*Prerequisite*: Graphene is set up and the nginx sample works.
+You can build the sample as follows:
+```sh
+export GRAPHENEDIR=[PATH To Your Graphene Folder]
+make
+```
 
-To marbleize the sample you must edit `nginx.manifest.template`. Please append or replace the following values:
+To marbleize the sample we edited `nginx.manifest.template`. Common things to change or add are:
```

Comment:
Maybe elaborate a bit on what certain changes mean and why they are necessary.
From a user's perspective, I'm more curious about that when I want to port my own project and not can't or don't want to just copy-paste here.

Reply:
I get your point on this. Thanks for the comments. I think this should go into the official  marblerun docs on how to "marblelize" your apps in general that run in Graphene or other mainstream LibOS implementations.

Then we can adjust this README, keep things small without losing 
concept and repeating ourselves. The sample is nice, not too complex and not too simple like a hello world and shows how we can achieve a synergy between our concepts and Graphene.